go/vt/tabletserver/config.go

```diff
@@ -32,6 +32,7 @@ var (
 func init() {
 	flag.IntVar(&qsConfig.PoolSize, "queryserver-config-pool-size", DefaultQsConfig.PoolSize, "query server connection pool size, connection pool is used by regular queries (non streaming, not in a transaction)")
 	flag.IntVar(&qsConfig.StreamPoolSize, "queryserver-config-stream-pool-size", DefaultQsConfig.StreamPoolSize, "query server stream pool size, stream pool is used by stream queries: queries that return results to client in a streaming fashion")
+	flag.IntVar(&qsConfig.MessagePoolSize, "queryserver-config-message-pool-size", DefaultQsConfig.MessagePoolSize, "query server message pool size, message pool is used by message managers: recommended value is one per message table")
```

Comment:
This description doesn't make it clear it's the connection pool to the DB we're talking about, especially with 'message pool size'. The previous one is not good either, we have:
- query server connection pool size (good!)
- query server stream pool size (not good, lost the 'connection')
- query server message pool size (not good, feels like it's a 'message pool' size).

Probably just re-add the 'connection' word before pool:
- query server connection pool size
- query server stream connection pool size
- query server message connection pool size